app/controllers/solidus_content/resource_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module SolidusContent
+  class ResourceController < Spree::Admin::ResourceController
+    private
+
+    def model_class
+      "SolidusContent::#{controller_name.classify}".constantize
+    end
+
+    def new_object_url(options = {})
+      new_admin_entry_type_path(options)
+    end
+
+    def edit_object_url(object, options = {})
+      edit_admin_entry_type_path(object, options)
+    end
+
+    def object_url(object = nil, options = {})
+      admin_entry_type_path((object || @object), options)
+    end
+
+    def collection_url(options = {})
+      admin_entry_types_path(options)
+    end
```

Comment:
Why do you need to override these?

Reply:
This is needed since the `Spree::ResourceController` uses the `Spree` namespace to build the resource URL and path helpers.

This extension uses the SolidusContent namespace instead.